spec/support/contexts/db_restore.rb

```diff
@@ -0,0 +1,96 @@
+require "active_record"
```

Comment:
Were we previously using active record in this gem? I wonder if we could get away with using PG instead?

Reply:
Barely, but yes:

https://github.com/ManageIQ/manageiq-gems-pending/blob/86396bfb/lib/gems/pending/util/extensions/miq-array.rb#L26

I am fine switching this to just using `PG` for these specs, in this section specifically, I knew the `ActiveRecord` interface better.

Reply:
Yeah, I think I would prefer PG here. Then if we extract this from gems pending we don't need to bring AR along.